pennylane/devices/default_qutrit_mixed.py

```diff
@@ -148,7 +148,7 @@ def f(x):
             program, execution_config = dev.preprocess()
             new_batch, post_processing_fn = program([qs])
             results = dev.execute(new_batch, execution_config=execution_config)
-            return post_processing_fn(results)
+            return post_processing_fn(results)[0]
```

Comment:
Without this, the result is a tuple `(res,)` instead of just `res`, and `jax.grad` below raises an error because it doesn't like the shape